PHPCompatibility/Tests/Classes/NewClassesUsesUnitTest.inc

```diff
@@ -0,0 +1,377 @@
+use SomeThingElse\IntlGregorianCalendar;
+use SomeThingElse\IntlTimeZone;
+use SomeThingElse\IntlBreakIterator;
+use SomeThingElse\IntlRuleBasedBreakIterator;
+use SomeThingElse\IntlCodePointBreakIterator;
+use SomeThingElse\IntlPartsIterator;
+use SomeThingElse\IntlIterator;
+use SomeThingElse\UConverter;
+use SomeThingElse\GMP;
+use SomeThingElse\IntlChar;
+use SomeThingElse\ReflectionType;
+use SomeThingElse\ReflectionGenerator;
+use SomeThingElse\ReflectionClassConstant;
+use SomeThingElse\ReflectionNamedType;
+use SomeThingElse\FFI;
+use SomeThingElse\FFI\CData;
+use SomeThingElse\FFI\CType;
+use SomeThingElse\ReflectionReference;
+use SomeThingElse\WeakReference;
+use SomeThingElse\PhpToken;
+use SomeThingElse\WeakMap;
+use SomeThingElse\ReflectionUnionType;
+use SomeThingElse\OCICollection;
+use SomeThingElse\OCILob;
+use SomeThingElse\Attribute;
+use SomeThingElse\IntlDatePatternGenerator;
+use SomeThingElse\Fiber;
+use SomeThingElse\ReflectionEnum;
+use SomeThingElse\ReflectionEnumBackedCase;
+use SomeThingElse\ReflectionEnumUnitCase;
+use SomeThingElse\ReflectionFiber;
+use SomeThingElse\ReflectionIntersectionType;
+use SomeThingElse\CURLStringFile;
+use SomeThingElse\Random\Randomizer;
+use SomeThingElse\Random\Engine\Mt19937;
+use SomeThingElse\Random\Engine\PcgOneseq128XslRr64;
+use SomeThingElse\Random\Engine\Xoshiro256StarStar;
+use SomeThingElse\Random\Engine\Secure;
+use SomeThingElse\DOMException;
+use SomeThingElse\Exception;
+use SomeThingElse\ReflectionException;
+use SomeThingElse\SoapFault;
+use SomeThingElse\SQLiteException;
+use SomeThingElse\ErrorException;
+use SomeThingElse\BadFunctionCallException;
+use SomeThingElse\BadMethodCallException;
+use SomeThingElse\DomainException;
+use SomeThingElse\InvalidArgumentException;
+use SomeThingElse\LengthException;
+use SomeThingElse\LogicException;
+use SomeThingElse\PDOException;
+use SomeThingElse\OutOfBoundsException;
+use SomeThingElse\OutOfRangeException;
+use SomeThingElse\OverflowException;
+use SomeThingElse\RangeException;
+use SomeThingElse\RuntimeException;
+use SomeThingElse\UnderflowException;
+use SomeThingElse\UnexpectedValueException;
+use SomeThingElse\SNMPException;
+use SomeThingElse\IntlException;
+use SomeThingElse\Error;
+use SomeThingElse\ArithmeticError;
+use SomeThingElse\AssertionError;
+use SomeThingElse\DivisionByZeroError;
+use SomeThingElse\ParseError;
+use SomeThingElse\TypeError;
+use SomeThingElse\ClosedGeneratorException;
+use SomeThingElse\ArgumentCountError;
+use SomeThingElse\HashContext;
+use SomeThingElse\SodiumException;
+use SomeThingElse\CompileError;
+use SomeThingElse\JsonException;
+use SomeThingElse\FFI\Exception;
+use SomeThingElse\FFI\ParserException;
+use SomeThingElse\UnhandledMatchError;
+use SomeThingElse\ValueError;
+use SomeThingElse\FiberError;
+use SomeThingElse\Random\RandomError;
+use SomeThingElse\Random\BrokenRandomEngineError;
+use SomeThingElse\Random\RandomException;
+
+new ArrayObject();
```

Comment:
I appreciate the effort which went into creating this test file.

the "class uses" code currently are all `new...` statements. Should this code be made a little more varied to safeguard that the protection against flagging imported classes works for all type of "class uses" that the sniff checks for ?

Reply:
I varied the the this test file a bit in lines 3-9 and 190-201. Do you have some. Do you have any other variations in mind in particular?

Reply:
This looks fine, Thanks for making those updates, including adding the group `use` and the case variations.